app/src/sharedTest/java/org/oppia/app/topic/questionplayer/QuestionPlayerActivityTest.kt

```diff
@@ -27,7 +27,7 @@ class QuestionPlayerActivityTest {
   @Test
   fun testQuestionPlayerActivity_loadQuestionPlayerFragment_hasDummyString() {
```

Comment:
Can we remove this testcase, if we aren't showing any dummy text.

Reply:
Good point. Replaced it with a temp test pointing to #503 to fill in actual functionality. I kept the test suite, though, as a placeholder for the future tests to go.

Reply:
Sounds reasonable.